docs/extras/integrations/llms/ctranslate2.ipynb

```diff
@@ -0,0 +1,167 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CTranslate2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**CTranslate2** is a C++ and Python library for efficient inference with Transformer models.\n",
+    "\n",
+    "The project implements a custom runtime that applies many performance optimization techniques such as weights quantization, layers fusion, batch reordering, etc., to accelerate and reduce the memory usage of Transformer models on CPU and GPU.\n",
+    "\n",
+    "Full list of features and supported models/frameworks is included in the [project's repository](https://github.com/OpenNMT/CTranslate2). To start, please check out the official [quickstart guide](https://opennmt.net/CTranslate2/quickstart.html).\n",
+    "\n",
+    "To use, you should have `ctranslate2` python package installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install ctranslate2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As explained in quickstart guide, to use Hugging Face model with CTranslate2, it has to be first converted to CTranslate2 format using the command `ct2-transformers-converter` command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!ct2-transformers-converter --model meta-llama/Llama-2-7b-hf --quantization int8 --output_dir llama-2-7b-ct2 --force"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.llms import CTranslate2\n",
+    "\n",
+    "llm = CTranslate2(\n",
+    "    model_path=\"/mnt/ml-team/homes/eryk.mazus/llama-2-7b-ct2\",\n",
```

Comment:
Do you want to leave your name here? It doesn't really matter, but I'm letting you know if you would like to keep it a secret :smile:

Reply:
TBH I think it would be beneficial to make it runnable - write how to download model and where & use it as path - so the notebook can be directly executed as tutorial.

Reply:
Fortunately, all you have to do, is to pass the name of the model from HF and it will be fetched without the need for manual work. I will uncomment the conversion command and use the same path in CTranslate2